nodewatcher/modules/devices/xiaomi/mi_router.py

```diff
@@ -0,0 +1,174 @@
+from django.utils.translation import ugettext_lazy as _
+
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class XiaomiMiMini(cgm_devices.DeviceBase):
+    """
+    Xiaomi Router Mini device descriptor.
+    """
+
+    identifier = 'mi-router-mini'
```

Comment:
I am not sure about the `mi-` prefix? So all Xiaomi devices will have this prefix? Or just Xiaomi Mi devices? Or are all Xiaomi devices called Mi?